From 0796c585972810f0b81acb82f479cd81d870170d Mon Sep 17 00:00:00 2001
From: Cristian Sovaiala <cristian.sovaiala@freescale.com>
Date: Fri, 4 Sep 2015 11:04:10 +0300
Subject: [PATCH 395/452] fsl-dpaa2: eth: Fix coccinelle issue

drivers/staging/fsl-dpaa2/ethernet/dpaa2-ethtool.c:687:1-36: WARNING:
Assignment of bool to 0/1

Signed-off-by: Cristian Sovaiala <cristi.sovaiala@nxp.com>
[Xulin: Original patch taken from FSL LS2085 SDK EAR6.0,
LS2085A-SDK-SOURCE-20160304-yocto.iso]
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 drivers/staging/fsl-dpaa2/ethernet/dpaa2-ethtool.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/staging/fsl-dpaa2/ethernet/dpaa2-ethtool.c b/drivers/staging/fsl-dpaa2/ethernet/dpaa2-ethtool.c
index 6dbfac6..2de87b6 100644
--- a/drivers/staging/fsl-dpaa2/ethernet/dpaa2-ethtool.c
+++ b/drivers/staging/fsl-dpaa2/ethernet/dpaa2-ethtool.c
@@ -700,7 +700,7 @@ static int dpaa2_do_cls(struct net_device *net_dev,
 	}
 
 	priv->cls_rule[fs->location].fs = *fs;
-	priv->cls_rule[fs->location].in_use = 1;
+	priv->cls_rule[fs->location].in_use = true;
 
 err_free_mem:
 	kfree(dma_mem);
-- 
1.7.5.4

